content/hardware/04.pro/shields/portenta-vision-shield/tutorials/speech-recognition-engine/content.md

```diff
@@ -193,7 +193,7 @@ Once everything is checked, click **Confirm** and you will get the model header
 
 You will now get some files in your e-mail inbox. Download them to your computer.
 
-On the IDE, open the example **File > Examples > Cyberon_DSpotterSDK > VoiceRecognition** and click **File > Save As...** and type a name for your sketch.
+On the IDE, open the example **File > Examples > DSpotterSDK_Maker_33BLE/NiclaVision/PortentaH7/RP2040 > VoiceRecognition** and click **File > Save As...** and type a name for your sketch.
```

Comment:
```suggestion
On the IDE, open the example **File -> Examples -> DSpotterSDK_<board_name> -> VoiceRecognition** and click **File -> Save As...** and type a name for your sketch.
```

Reply:
Thanks for the suggestion. I have now made the suggested change: [`28ffb82` (#1643)](https://github.com/arduino/docs-content/pull/1643/commits/28ffb82f03847bc6a0df7a5c4a92f4c4f871551c)